### DISCUSSION TITLE
Wallpaper

### DISCUSSION THREAD

**Discussion by normrioux** [SafeExamBrowser/seb-win-refactoring#402] [General]
Hello, 

I would like to know if it's possible to configure a background picture ("Wallpaper") instead of a black screen in SEB. 

My goal is to see at a glance if the students are in the right configuration. (Exams are mostly in computer labs) I'm using two type of exams config; with a browser and with applications only.

Thanks!

Normand

**Comment by dbuechel** [comment]
Unfortunately, there is no such configuration option at the moment. Please feel free to create a feature request as an issue, I'd however need to advert that our current backlog is already full for at least the upcoming year, so it might take quite a while until we'd get around to hazard an implementation attempt. If you're member of an institution however, you may want to consider a [SEB Alliance membership](https://safeexambrowser.org/alliance/members.html) which could potentially speed up the process.

**Comment by normrioux** [comment]
Thanks a lot for your quick answer! No need for feature request as it's a bit specific to our needs... But I will keep in mind the SEB Alliance membership if needed. Thanks!

**Reply by dbuechel** [reply]
Excellent, you're most welcome
